src/store/user-store.js

```diff
@@ -11,6 +13,18 @@ class UserStore {
   setIsLoggedIn(value) {
     this.isLoggedIn = value;
   }
+
+  setUserResponseData(data) {
+    this.userResponse = data;
+  }
```

Comment:
Step 3: We need a setter function that will take in data, and set the value as `this.userResponse`. 

You can learn more about `getters` and `setters` [here](https://www.programiz.com/javascript/getter-setter)